learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+
+class TestCreateCourse(TestCase):
+    """
+    Test course creation API.
+    """
+    def setUp(self):
+        """
+        Set some test data
+        """
+        super(TestCreateCourse, self).setUp()
+        self.user, _ = User.objects.get_or_create(username="test")
```

Comment:
The tuple unpacking here should be removed (forgive me if you're already aware of this or see this many times in this code review, just being cautious)